roles/ceph-mon/tasks/docker/start_docker_monitor.yml

```diff
@@ -1,4 +1,7 @@
 ---
+- name: pull ceph daemon image
+  shell: "docker pull {{ ceph_mon_docker_username }}/{{ ceph_mon_docker_imagename }}"
```

Comment:
that's right, `docker run` will pull the image if it is not there. But on the VM, without pulling image first, `dockr run` always fail. Could be timeout? Not sure.